pyogrio/geopandas.py

```diff
@@ -16,6 +16,7 @@ def read_dataframe(
     where=None,
     bbox=None,
     fids=None,
+    set_fid_index=False,
```

Comment:
I find this keyword a bit cryptic at first sight. What about `fid_as_index` instead?